themes/default/layouts/page/automation.html

```diff
@@ -3,193 +3,422 @@
 {{ end }}
 
 {{ define "main" }}
-    <section id="overview" class="container mx-auto text-center my-16">
+    <section id="overview" class="container mx-auto my-16 lg:mt-16 lg:mb-8 px-6">
         <div class="lg:flex">
-            <div class="lg:w-1/2 px-16">
-                <h3>What if infrastructure as code was just a library?</h3>
+            <div class="lg:w-1/2 lg:order-last mb-8 lg:mb-0 relative h-full">
+                <div class="shape-background automation-video">
+                    <div class="shape-container">
+                        <div class="circle-left"></div>
+                        <div class="circle-right"></div>
+                    </div>
+                </div>
+
+                <div class="rounded-xl shadow-2xl relative h-0 overflow-hidden" style="padding-bottom: 50.25%;">
+                    <iframe
+                        src="//www.youtube.com/embed/kFHb8f2fk9k?rel=0"
+                        style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; border:0;"
+                        allowfullscreen=""
+                        title="Automation API Overview"
+                        srcdoc="<style>*{padding:0;margin:0;overflow:hidden}html,body{height:100%}img{position:absolute;width:100%;top:0;bottom:0;margin:auto}</style><a href=https://www.youtube.com/embed/kFHb8f2fk9k?autoplay=1><img src='/images/video-cards/automation-api-demo.png' alt='Introduction to Pulumi: Modern Infrastructure as Code'></a>"
+                    ></iframe>
+                </div>
+            </div>
+            <div class="lg:w-1/2 lg:pr-16 text-center">
+                <h3>Harness the full power of Infrastructure as Code</h3>
                 <p>
-                    Build custom cloud tools, frameworks, self-serve portals, and more,
-                    right from your favorite programming language.
+                    Pulumi Automation API exposes the full power of infrastructure as code through a
```

Comment:
" Pulumi Automation API exposes the full power of infrastructure as code through a
                     programmatic interface, instead of through CLI commands. Automation API lets you
                     use the Pulumi engine as an SDK, enabling you to create software that can create, update, configure, and destroy
                     infrastructure dynamically. This enables you to build custom cloud interfaces that are
                     tailored to your team,  organization, or customers."

Reply:
could you make this paragraph left justified? it looks funny with the current formatting

Reply:
👍

Reply:
for the video, could we make this graphic the thumbnail? (but add a play button so that people know to click it?)

![](https://www.pulumi.com/images/docs/reference/automation-api.png)